src/raftstore/store/engine.rs

```diff
@@ -81,6 +81,23 @@ impl Snapshot {
     pub fn get_db(&self) -> Arc<DB> {
         self.db.clone()
     }
+
+    pub fn new_arc_iter(&self, iter_opt: IterOption) -> DBIterator<Arc<DB>> {
```

Comment:
why not using new_iterator directly?

Reply:
Because `Iterable` needs `&DB`, `Arc<DB>` won't be acceped.

Reply:
`arc_iter` seems to be an ambiguous name.